llama_index/program/llm_program.py

```diff
@@ -47,16 +50,24 @@ def from_defaults(
             raise ValueError("Must provide either prompt or prompt_template_str.")
         if prompt_template_str is not None:
             prompt = PromptTemplate(prompt_template_str)
+
+        # decide default output class if not set
+        if output_cls is None:
```

Comment:
much needed!

Reply:
Should it just default to a pydantic output parser if not provided?

Reply:
you mean if output_cls is not None but output parser is?

Reply:
Yea. That would be more similar to the openai pydantic program. Less imprts for the user